appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/cmd/options/RuntimeOptions.java

```diff
@@ -49,7 +59,7 @@
  */
 public class RuntimeOptions {
 
-    private Map<RUNTIME_OPTION, List<String>> options;
+    private final List<Map.Entry<RUNTIME_OPTION, String>> options;
```

Comment:
I think you're loooking for `LinkedHashMap` here.

Reply:
I think, no. When deploy options is mixed, e.g. --deploy --deployFromGAV --deploy, we cannot preserve they deployment order in case of  linked hash map.